apps/taxes/graphql/fragments/TaxBase.graphql

```diff
@@ -61,11 +61,19 @@ fragment TaxBase on TaxableObject {
   }
   sourceObject {
     ... on Checkout {
+      metadata {
+        key
+        value
+      }
```

Comment:
you dont have to fetch all metadata, you can use

```
query {
  order{
    avataxEntityCode: metafield(key:"avataxEntityCode")
  }
}
```

Reply:
Cool, I didn't know that. Refactored it